README.rst

```diff
@@ -104,22 +105,22 @@ You can run Fluent TUI commands using the ``session.tui`` interface:
   session.solver.tui.solve.initialize.initialize_flow()
   session.solver.tui.solve.dual_time_iterate(2, 3)
 
-In addition to all TUI commands being available there are the
+In addition to all TUI commands being available, there are the
 `PyFluent Parametric <https://fluentparametric.docs.pyansys.com/>`_ and
 `PyFluent Visualization <https://fluentvisualization.docs.pyansys.com/>`_ packages.
-The PyFluent Parametric package provides access to Fluent's parametric workflows and
-the PyFluent Visualization package provides post-processing and visualization
-capabilities using `pyvista <https://docs.pyvista.org/>`_ and
-`matplotlib <https://matplotlib.org/>`_.
 
-License and Acknowledgments
+- The PyFluent Parametric package provides access to Fluent's parametric workflows.
```

Comment:
Kathy, FWIW I usually use rewrap VS code plugin set at 80 columns. Some of the content looks to be more than 80.